PMSuperButtonSample/Base.lproj/Main.storyboard

```diff
@@ -190,6 +213,7 @@
     </scenes>
     <resources>
         <image name="background" width="1280" height="1280"/>
+        <image name="glyphicons-basic-790-fish" width="32" height="32"/>
```

Comment:
maybe u should change the name of the image. please set a name like the other ones.